gains:
  shoulder_pan_joint: {p: 10000, d: 100, i: 1000, i_clam: 1.0}
  shoulder_lift_joint: {p: 10000, d: 100, i: 1000, i_clam: 1.0}
  elbow_joint: {p: 10000, d: 100, i: 1000, i_clam: 1.0}
  wrist_1_joint: {p: 1000, d: 10, i: 0, i_clam: 1.0}
  wrist_2_joint: {p: 1000, d: 10, i: 0, i_clam: 1.0}
  wrist_3_joint: {p: 1000, d: 1, i: 0, i_clam: 1.0}

velocity:
  pid_gains:
    shoulder_pan_joint: {p: 30, d: 0, i: 0, i_clam: 1.0}
    shoulder_lift_joint: {p: 30, d: 0, i: 0, i_clam: 1.0}
    elbow_joint: {p: 30, d: 0, i: 0, i_clam: 1.0}
    wrist_1_joint: {p: 30, d: 0, i: 0, i_clam: 1.0}
    wrist_2_joint: {p: 30, d: 0, i: 0, i_clam: 1.0}
    wrist_3_joint: {p: 30, d: 0, i: 0, i_clam: 1.0}
